activities/index.md

```diff
@@ -19,6 +19,7 @@ Activities that support the above as well as make staff operations work:
 * [Conflicts of interest](conflict-of-interest/index.md)
 * [Creating affiliations](creating-affiliations/index.md)
 * [Creating partnerships](creating-partnerships/index.md)
+* [Codebase stewardship](codebase-stewardship/index.md)
```

Comment:
Indeed this is also above on line 11, pulled out special.

It's not _wrong_ to say that Codebase stewardship supports Codebase stewardship, although it feels a bit silly.

I suppose maybe the more important question is whether or not we'd miss it in the bullet list if we went looking for it.